### PR TITLE
[plugin-excel] Add ability to preserve cells format

### PR DESCRIPTION
**Pull request by draker94**
Related to #4553

If we want to add cell typing it makes sense add ability to preserve cells format, otherwise, for example, dates, will be displayed as a number, and all numbers themselves will be displayed in fractional format no matter with which style we created the cell

### CODE REVIEW

File: vividus-plugin-excel/src/main/java/org/vividus/excel/transformer/ExcelTableTransformer.java [hunk 1]
```diff
@@ -127,4 +128,9 @@ private List<String> replaceLineBreaks(List<String> list, String lineBreakReplac
                      .map(e -> e.replace("\n", lineBreakReplacement))
                      .toList();
     }
+
+    public void setPreserveCellFormatting(boolean preserveCellFormatting)
```
Comment (valfirst) [inline vividus-plugin-excel/src/main/java/org/vividus/excel/transformer/ExcelTableTransformer.java:132]: please use constructor to set properties

File: vividus-plugin-excel/src/main/java/org/vividus/steps/ExcelDocumentValidationSteps.java [hunk 2]
```diff
@@ -162,4 +163,9 @@ private static Predicate<Entry<CellValue, Optional<Pattern>>> filterMatched()
             return pattern.isPresent() || cellValue.value() != null;
         };
     }
+
+    public void setPreserveCellFormatting(boolean preserveCellFormatting)
```
Comment (valfirst) [inline vividus-plugin-excel/src/main/java/org/vividus/steps/ExcelDocumentValidationSteps.java:167]: please use constructor to set properties

File: vividus-tests/src/main/resources/story/integration/TableTransformers.story [hunk 3]
```diff
@@ -309,20 +309,6 @@ Then `${resultSequenceOfTransformers}` is equal to table:
...
-    @issueId 2451
-Then `<relativeUrl>` matches `.*links.*`
-Examples:
-{transformer=FROM_HEADLESS_CRAWLING, column=relativeUrl}
-
-
-Scenario: Verify FROM_HEADLESS_CRAWLING transformer passing URL through transformer parameter
-Then `<relativeUrl>` matches `.*links.*`
-Examples:
-{transformer=FROM_HEADLESS_CRAWLING, column=relativeUrl, mainPageUrl=$\{vividus-test-site-url\}}
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/TableTransformers.story:323]: Why are these scenarios removed?

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-tests/src/main/resources/story/integration/TableTransformers.story (hunk 3 above)
Comment (draker94) [reply vividus-tests/src/main/resources/story/integration/TableTransformers.story]: Forgot to restore after local changes.

**Review by valfirst — APPROVED** [review]

**Pull request merged**